src/core/stdc/fenv.d

```diff
@@ -783,11 +790,20 @@ else
             FE_TOWARDZERO   = 0x1, ///
         }
     }
+    else version (WebAssembly) // TODO: needs to be WASI
```

Comment:
I think you can add the `WASI` identifier using this PR (https://github.com/dlang/dmd/pull/7202) as an example.